### PR TITLE
feat(ELB): add new attributes

### PR DESCRIPTION
**Pull request by 346010762**
**What this PR does / why we need it**:
elb adds new attributes
**Special notes for your reviewer**:

**Release note**:

```release-note
elb adds new attributes
```

## PR Checklist

* [ ] Tests added/passed.
* [ ] Documentation updated.
* [ ] Schema updated.

## Acceptance Steps Performed

```
make testacc TEST=./huaweicloud/services/acceptance/elb/ TESTARGS='-run TestAccElbV3LoadBalancer'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud/services/acceptance/elb/ -v -run TestAccElbV3LoadBalancer -timeout 360m -parallel 4
=== RUN   TestAccElbV3LoadBalancer_basic
=== PAUSE TestAccElbV3LoadBalancer_basic
=== RUN   TestAccElbV3LoadBalancer_withEpsId
=== PAUSE TestAccElbV3LoadBalancer_withEpsId
=== RUN   TestAccElbV3LoadBalancer_withEIP
=== PAUSE TestAccElbV3LoadBalancer_withEIP
=== RUN   TestAccElbV3LoadBalancer_withEIP_Bandwidth_Id
=== PAUSE TestAccElbV3LoadBalancer_withEIP_Bandwidth_Id
=== RUN   TestAccElbV3LoadBalancer_prePaid
=== PAUSE TestAccElbV3LoadBalancer_prePaid
=== RUN   TestAccElbV3LoadBalancer_availabilityZone
=== PAUSE TestAccElbV3LoadBalancer_availabilityZone
=== CONT  TestAccElbV3LoadBalancer_basic
=== CONT  TestAccElbV3LoadBalancer_availabilityZone
=== CONT  TestAccElbV3LoadBalancer_prePaid
=== CONT  TestAccElbV3LoadBalancer_withEIP_Bandwidth_Id
--- PASS: TestAccElbV3LoadBalancer_withEIP_Bandwidth_Id (71.61s)
=== CONT  TestAccElbV3LoadBalancer_withEIP
--- PASS: TestAccElbV3LoadBalancer_withEIP (53.18s)
=== CONT  TestAccElbV3LoadBalancer_withEpsId
--- PASS: TestAccElbV3LoadBalancer_withEpsId (47.39s)
--- PASS: TestAccElbV3LoadBalancer_availabilityZone (172.28s)
--- PASS: TestAccElbV3LoadBalancer_basic (179.83s)
--- PASS: TestAccElbV3LoadBalancer_prePaid (287.70s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/elb       287.760s

```

### CODE REVIEW

File: docs/resources/elb_loadbalancer.md [hunk 1]
```diff
@@ -195,6 +195,16 @@ The following arguments are supported:
...
 
+* `deletion_protection_enable` - (Optional, Bool) Specifies whether to enable deletion protection
+  for the load balancer. Value options:
+  + **true**: Enable deletion protection.
+  + **false**: Disable deletion protection.
+
+* `waf_failure_action` - (Optional, String) Specifies traffic distributing policies when the WAF is faulty.
+  Value options:
+  + **discard**: Traffic will not be distributed.
+  + **false**: Traffic will be distributed to the default backend servers.
```
Comment (houpeng80) [inline docs/resources/elb_loadbalancer.md]:
```suggestion
  + **forward**: Traffic will be distributed to the default backend servers.
```

File: docs/resources/elb_loadbalancer.md [hunk 2]
```diff
@@ -234,6 +268,7 @@ resource "huaweicloud_elb_loadbalancer" "loadbalancer_1" {
   lifecycle {
     ignore_changes = [
       ipv6_bandwidth_id, iptype, bandwidth_charge_mode, sharetype, bandwidth_size, bandwidth_id, force_delete,
+      deletion_protection_enable
```
Comment (houpeng80) [inline docs/resources/elb_loadbalancer.md]:
```suggestion
      deletion_protection_enable,
```

File: docs/resources/elb_loadbalancer.md (hunk 1 above)
Comment (346010762) [reply docs/resources/elb_loadbalancer.md]: done

File: docs/resources/elb_loadbalancer.md (hunk 2 above)
Comment (346010762) [reply docs/resources/elb_loadbalancer.md]: done

**Pull request merged**